### PR TITLE
disable save draft where appropriate

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

**Review by stuartromanek — APPROVED** [review]
I made a follow up ticket for proper disabling of context menu items https://linear.app/apostrophecms/issue/PRO-1326/context-menu-items-should-be-disable-able

File: modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue [hunk 1]
```diff
@@ -223,6 +224,21 @@ export default {
...
+    // Whether to show the feature at all
+    canSaveDraft() {
+      return this.manuallyPublished;
+    },
+    // Whether to disable it when it is shown
+    disableSaveDraft() {
+      if (!this.published) {
+        return false;
+      }
+      // There must be a new change since you opened the modal
```
Comment (abea) [inline modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue:236]: I'm confused by this. I thought that was the problem.

File: modules/@apostrophecms/doc-type/ui/apos/components/AposDocMoreMenu.vue [hunk 2]
```diff
@@ -114,6 +120,9 @@ export default {
     },
     canDismissSubmission() {
       this.menu = this.recomputeMenu();
+    },
+    disableSaveDraft() {
```
Comment (abea) [inline modules/@apostrophecms/doc-type/ui/apos/components/AposDocMoreMenu.vue:124]: Why do we need a new prop and watcher if we're already watching `isPublished` and `isModified`?

**Review by abea — CHANGES_REQUESTED** [review]

**Comment by boutell** [comment]
Closing in favor of PRO-1304, if any further adjustment is needed it'll look different.

**Pull request closed without merging**